src/components/header/menu.tsx

```diff
@@ -0,0 +1,135 @@
+import { Link } from "react-router-dom"
+import  SteamLogo from './../../assets/SteamLogo.svg'
+import Authorization from "../Action/authorization"
+import { useState } from "react"
+import useLogin from "../../hooks/useLogin"
+
+type MenuItem = {
+    name: string;
+    link: string;
+    dropDown?: MenuItem[];
+};
+
+const Menu = () =>{
+    const [activeStyle, setActiveStyle] = useState('STORE')
+    const { data: user} = useLogin();
+
+    const MOCK_MENU: MenuItem[] = [
+        {

```

Comment:
also you won't need most of the links for now. so let's clean the menu items
Let's leave only the following:
STORE - Home, Wishlist
COMMUNITY - remove it fully
PROFILE - Activity, Profile, Friends